docs/Introduction.md

```diff
@@ -1,21 +1,21 @@
 # Introduction
-**Subscription Management** - a turnkey solution that provides an easy and secure monetization mechanism for your app. Backed by Stripe, it provides an automated and integrated solution to manage your customers, subscriptions and handle payments.
+**Subscription Management** - is a turnkey solution that provides an easy and secure monetization mechanism for your Business Central app. Designed by Theta and backed by Stripe, it provides an automated and integrated solution to handle payments and manage your customers and subscriptions.
 
 ## Key Features 
-- **Backed by Stripe**, #1 payment platform available in all BC SaaS countires.
-- Offer **flexible billing models** including tiered, per seat and usage based pricing.
-- Supports **135+ currencies** and multiple payment intervals.
-- No middle tier required. Only **direct**, **secure**, end-to-end **integration** with app owner’s Stripe account.
+- **Backed by Stripe**, the #1 payment platform available in all Business Central SaaS countries.
+- Offers **flexible billing models** including tiered, per seat and usage based pricing. Any model you choose to provide your customers.
+- Supports **135+ currencies** and multiple payment intervals. You select the currencies and payment intervals you accept.
+- Only **direct**, **secure**, **end-to-end** integration with the app publishers own Stripe account. No middle tier required.
```

Comment:
We may need to go away from "publisher" to "vendor" as app allows not only AppSource publishers to use it, but for a ISVs which provides apps for on-prem environment.

Reply:
OK, we need to choose one and apply it consistently.

Reply:
After hour of reading, my ranking is:
1. publisher
2. maker
3. ISV
Pick one and let's go with it.